### ISSUE TITLE
fail to run dpdk_iface_main and dpdk0 don't appear

### ISSUE THREAD

**Issue by kwjjyn** [mtcp-stack/mtcp#254]
hi, mtcp team.
I'm using virtio driver to run the mtcp . However, it seems that it would compile dpdk-iface-kmod only using intel drivers. I tried to compile the dpdk-iface-kmod but it failed. So I can't get the dpdk0 interface and can't assign ip on it . Could you give me some advices ? Thank you very much.

**Comment by ajamshed** [comment]
@kwjjyn,

As of this moment, we don't provide support for `virtio` driver. Can you switch to `igb_uio` driver?

**Comment by kwjjyn** [comment]
yes , I can use dpdk to switch to igb_uio driver. Actually the error is :
error: unknown field ‘ndo_setup_tc’ specified in initializer
         .ndo_setup_tc           = NULL
it seems the kernel header problem.   The kernel version I used is 3.10.0 .  And I found the function ndo_setuo_tc was changed to nod_setup_tc_rh72 in netdevice.h.
After changing the function name in dpdk_iface.h , the dpdk0 shows up

**Comment by ajamshed** [comment]
Thanks for reporting the problem. We started providing support for mTCP-DPDK from `linux-3.13`. But I think you have patched up the code well.
